random-beans/src/main/java/io/github/benas/randombeans/util/Constants.java

```diff
@@ -34,37 +34,34 @@
  *
  * @author Mahmoud Ben Hassine (mahmoud.benhassine@icloud.com)
  */
-public abstract class Constants {
```

Comment:
I think this should stay an abstract class (constant interfaces are a bit of an anti-patter see e.g. Effective Jave "Item 19: Use interfaces only to define types").